lib/Notification/styles.scss

```diff
@@ -1,9 +1,7 @@
 .notification {
-	font-size: 13px;
-	line-height: 1.75em;
-	padding: 10px 15px;
+	font-size: $typo-size-slight;
+	padding: $layout-spacing-base/2 $layout-spacing-base*0.75;
```

Comment:
@magalhini I've got a great idea about how we can handle this moving forward. Going to propose something in the PR related to progress.